www/frontend/index.html

```diff
@@ -0,0 +1,129 @@
+        <br/>
+        <script>
+            
+            let submit = document.getElementById('submitButton');
+            submit.addEventListener('click', function(event) {
+                event.preventDefault();
+                let ul = $('#list');
+                let taskToBeCreated = $('#newTask');
+                const data = JSON.stringify({
+                    task: taskToBeCreated.val()
+                });
+
+                $.ajax({
+                    async: true,
+                    crossDomain: true,
+                    contentType: 'application/json; charet=utf-8',
+                    processData: false,
+                    method: "POST",
+                    url: "http://localhost/backend-api/router.php",
+                    data: data,
+                    dataType: "json",
+                    success: function(data) {
+                        alert(data.message);
+                        let li = document.createElement("li");
+                        li.setAttribute('id', 'taskItem_'+data.message);
+                        li.appendChild(document.createTextNode(data.message));
+                        let deleteButton = document.createElement("button");
+                        deleteButton.setAttribute('id', 'task_'+ data.message);
+                        deleteButton.setAttribute('onclick','deleteTask();'); // for FF
+                        deleteButton.onclick = function() { deleteTask(data.message); }; // for IE
+                        deleteButton.innerText = "Delete";
+
+                        let updateButton = document.createElement("button");
+                        updateButton.setAttribute('id', 'task_'+ data.message);
+                        updateButton.setAttribute('onclick','updateTask();'); // for FF
+                        updateButton.onclick = function() { updateTask(data.message); }; // for IE
+                        updateButton.innerText = "Update";
+
+                        li.append(deleteButton);
+                        li.append(updateButton);
+                        ul.append(li);  
+                    },
+                    failure: function(error) {
+                        alert("this is an error" + error);
+                    }
+                });
+            });
+           
+            function getTask() {
+               
+                $.ajax({
+                    method: "GET",
+                    url: "http://localhost/backend-api/router.php",
+                    success: function(data) {
+                      alert(data);
+
+                    },
+                    failure: function(error) {
+                        alert("This is an error: " + error);
+                    }
+                });
+            }
+
+            function updateTask(task) {
+                let newTask = prompt('Insert new task');
+                const data = JSON.stringify({
+                    oldTask: task,
+                    newTask: newTask
```

Comment:
Any reason to send the previous version of the task along with the new? Seems like it just makes for a larger request body.